### PR TITLE
add grunt task

### PR DESCRIPTION
**Pull request by patrickkettner**

### CODE REVIEW

File: tasks/mock-server.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    // The HTTPS port for the server
+    // Defaults to 8081
+    config.securePort = parseInt(options.securePort || config.isStandalone ? 443 : 8081);
+
+    if (!config.appDir) {
+      throw grunt.util.error('"appDir" option required for mock-server task');
+    }
+
+    require('../lib');
```
Comment (kpdecker) [inline tasks/mock-server.js:62]: What is this doing? Starting the app?

Comment (patrickkettner) [inline tasks/mock-server.js:62]: yerp

Comment (kpdecker) [inline tasks/mock-server.js:62]: Ugh, thats kind of a crappy API. Could you add a quick comment saying that
is what this is doing and then we should be good?

Comment (patrickkettner) [inline tasks/mock-server.js:62]: yeah, its a weird one. I [got it from the cli](https://github.com/walmartlabs/mock-server/blob/master/bin/mock-server). Would you like to make that API more explicit?

Comment (kpdecker) [inline tasks/mock-server.js:62]: Longer term that would be nice but for the near term a comment like "start
the process" would probably be sufficent.

On Sunday, December 22, 2013, patrick kettner wrote:

> In tasks/mock-server.js:
> 
> > -    // Defaults to 8080
> > -    config.port = parseInt(options.port || process.env.PORT || 8080);
> >   +
> > -    // The HTTP port for the server if you are behind a proxy.
> > -    // Defaults to 8080
> > -    config.forwardPort = parseInt(options.port || config.isStandalone ? 80 : config.port);
> >   +
> > -    // The HTTPS port for the server
> > -    // Defaults to 8081
> > -    config.securePort = parseInt(options.securePort || config.isStandalone ? 443 : 8081);
> >   +
> > -    if (!config.appDir) {
> > -      throw grunt.util.error('"appDir" option required for mock-server task');
> > -    }
> >   +
> > -    require('../lib');
> 
> yeah, its a weird one. I got it from the clihttps://github.com/walmartlabs/mock-server/blob/master/bin/mock-server.
> Would you like to make that API more explicit?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/walmartlabs/mock-server/pull/15/files#r8519528
> .

## 

Sent from my mobile device

Comment (patrickkettner) [inline tasks/mock-server.js:62]: already done

**Comment by kpdecker** [comment]
Released in 1.3.0

**Pull request merged**